### DISCUSSION TITLE
Slashcommand for going to the currently active update set

### DISCUSSION THREAD

**Discussion by VanWeapon** [arnoudkooi/SN-Utils#412] [Ideas]
I've been trying to figure out how to open the currently selected update set via a slashcommand, but I cant find anything in the DOM or browser console that I can call to figure this out, but surely there's something otherwise the update set picker wouldn't know what set is active. 

I'm fine with creating a custom command for this but just can't find what to call. 

Outcome is I can use `/us` or something to go to the active set and complete it without having to click twice through the picker

**Comment by arnoudkooi** [comment]
I think with URL:  `sys_update_set.do?sys_id=javascript:gs.getPreference('sys_update_set')` it should work!

**Comment by VanWeapon** [comment]
It works! Many thanks

**Reply by arnoudkooi** [reply]
Great!
